src/streams.c

```diff
@@ -10,8 +10,6 @@
 #include <time.h>
 #include <signal.h>
 #include <fcntl.h>
-#include <spawn.h>
-#include <sys/wait.h>
```

Comment:
These two headers are included below in the `#if !defined(_WIN32) && !defined(__wasi__)` bit, BTW.
Which reminds me, I should add Windows to the CI. I'll take care of that later :-)